### DISCUSSION TITLE
OpenID Connect does not work

### DISCUSSION THREAD

**Discussion by nguyentruongtho** [saleor/saleor#9948] [Q&A] [answered]
I am trying to follow the instruction here https://docs.saleor.io/docs/3.x/developer/available-plugins/openid-connect but I get this error of correct input:

<img width="1551" alt="image" src="https://user-images.githubusercontent.com/1457567/172581334-3ac55914-8bdc-486b-9372-2b2f82260de1.png">

**Comment by timuric** [comment, marked as answer]
@nguyentruongtho please provide input as a variable 

**mutation:**
```
mutation ExternalAuthenticationUrl($input: JSONString!){
  externalAuthenticationUrl(
    pluginId:"mirumee.authentication.openidconnect", 
    input: $input
  ){
    authenticationData
    accountErrors{
      field
      message
    }
  }
}
```

**variables**
```
{
	"input": "{\"redirectUri\":\"http://127.0.0.1:3001/callback\"}"
}
```

Thanks for reporting we will update the docs.

**Reply by timuric** [reply]
PR https://github.com/saleor/saleor-docs/pull/440
